### PR TITLE
Added playlist details page

### PR DESCRIPTION
**Pull request by sanjitsarkar**
**In this PR,**
I have added playlist info page.
Preview Link [Playlist Info Page](https://deploy-preview-8--zentube.netlify.app/)
**Things to check**
- [ ] PlaylistDetailsPage

Things not to check
- [ ] Remove from playlist button on Playlist info page

### CODE REVIEW

**Review by rutu14 — APPROVED** [review]
The code is good to merge.
UI is amazing.

File: src/pages/PlaylistDetailsPage/PlaylistInfo.jsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        {playlistsInfo.get(playlistId).title}
+      </h3>
+      <p className="o-60 ml-1 text-md">
+        {playlistsInfo.get(playlistId).description}
+      </p>
+    </div>
+  );
+};
+
+export default PlaylistInfo;
```
Comment (deekshithmd) [inline src/pages/PlaylistDetailsPage/PlaylistInfo.jsx:21]: Code looks  good, you can proceed to merge

**Review by deekshithmd — APPROVED** [review]

**Review by anshulraheja — APPROVED** [review]
Well written code, you can merge

**Review by manish-GH — APPROVED** [review]
The code is easy to read and the UI is great. Feel free to merge

**Pull request merged**